recipes/labbench/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "labbench" %}
+{% set version = "0.37.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/labbench-{{ version }}.tar.gz
+  sha256: 922a38307906bfca192845ad6b40b65bdf3db8b2ddc35ac133b36072ecf249fe
+
+build:
+  skip: True  # [py >= 313]
```

Comment:
Please exclude this. Python 3.13 support will be added via a migration later this year. If you have this line, it will never be attempted.
```suggestion
```